packages/block-editor/src/store/reducer.js

```diff
@@ -983,6 +983,32 @@ export const blockListSettings = ( state = {}, action ) => {
 	return state;
 };
 
+/**
+ * Reducer return an updated state representing the most recent block attribute
+ * update. The state is structured as a tuple of the clientId of the block and
+ * the partial object of updated attributes values.
+ *
+ * @param {[string,Object]} state  Current state.
+ * @param {Object}          action Action object.
+ *
+ * @return {[string,Object]} Updated state.
+ */
+export function lastBlockAttributesChange( state = [ null, null ], action ) {
```

Comment:
> Is this exported intentionally?

It would be needed for unit tests, once I've implemented them 😄 

Despite being exported from the file, none of these reducers would be publicly available on the top-level API of the module.